.github/workflows/codespell.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: Codespell
```

Comment:
Is it possible to configure exceptions? Otherwise this job will always fail because it detects false positives.

Reply:
yes, there is a configuration added to [pyproject.toml](https://github.com/go-gitea/gitea/pull/26194/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711) where I already configured many exceptions: https://github.com/go-gitea/gitea/pull/26194/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R16 .